content/d3/concepts/interactivity/terms/on/on.md

```diff
@@ -0,0 +1,41 @@
+---
+Title: '.on()'
+Description: 'Adds a specific event handler to an element.'
+Subjects:
+  - 'Data Science'
+  - 'Data Visualization'
+Tags:
+  - 'Graphs'
+  - 'Libraries'
+  - 'Events'
+  - 'DOM'
+CatalogContent:
+  - 'learn-d3'
+  - 'paths/data-science'
+---
+
+In **D3.js**, the **d3.selection.on()** function is used to add a specific event handler to an element. An event can be any Document Object Model (DOM) event type such as `click`, `zoom`, `mouseenter`, `mouseleave`, `mouseover`, `mouseout`, and `mousemove`.
+
+## Syntax
+
+```pseudo
+selection.on(event type, callback function)
+```
+
+- `event type`: The **event type** is a *string* that specifies the DOM event type used.
+- `callback function`: The [callback function](https://www.codecademy.com/resources/docs/javascript/callbacks) is the *function* that is called when the event is triggered.
```

Comment:
```suggestion
- `callback function`: The [callback function](https://www.codecademy.com/resources/docs/javascript/callbacks) is the function that is called when the event is triggered.
```

Reply:
Got it. I will add all the suggested changes as soon as possible.